packages/components/src/Components/Truncate/Truncate.js

```diff
@@ -1,13 +1,12 @@
-import React from 'react';
-import propTypes from 'prop-types';
-
-import classNames from 'classnames';
-
-import { Button, Stack, StackItem } from '@patternfly/react-core';
-
 import './truncate.scss';
 
-import sanitizeHtml from 'sanitize-html';
+import { Stack, StackItem } from '@patternfly/react-core/dist/js/layouts/Stack/index';
```

Comment:
`/js/layouts/Stack/Stack'` index has all components from `/Stack` folder (dunno if there is more)

Reply:
? confused, yeah we're using the index file here?

Reply:
You want to use the actual file with the component. The index is only exporting all files from the same folder so you might end up with more components in your build. Like with `Button/Button` you want to point towards `Stack/Stack`